release-please-config.json

```diff
@@ -9,11 +9,9 @@
       "component": "puppeteer-core"
     },
     "packages/testserver": {},
-    "packages/ng-schematics": {
-      "initial-version": "0.1.0"
-    },
+    "packages/ng-schematics": {},
     "packages/browsers": {
-      "initial-version": "0.0.1"
+      "release-as": "0.0.4"
```

Comment:
this appears to be the wrong version. The next version is `0.0.5`, `0.0.4` is already published. Why was this change needed?